src/dali/plugin/input/rest/binance_com.py

```diff
@@ -0,0 +1,1295 @@
+                    asset=transaction[_ASSET],
+                    exchange=self.__BINANCE_COM,
+                    holder=self.account_holder,
+                    transaction_type=transaction_type.value,
+                    spot_price=Keyword.UNKNOWN.value,
+                    crypto_in=str(amount),
+                    crypto_fee=None,
+                    fiat_in_no_fee=None,
+                    fiat_in_with_fee=None,
+                    fiat_fee=None,
+                    notes=notes,
+                )
+            )
+
+    def _process_sell(self, transaction: Any, out_transaction_list: List[OutTransaction], notes: Optional[str] = None) -> None:
+        trade: _Trade = self._to_trade(transaction[_SYMBOL], str(transaction[_AMOUNT]), str(transaction[_COST]))
+
+        # For some reason CCXT outputs amounts in float
+        if transaction[_SIDE] == _BUY:
+            out_asset = trade.quote_asset
+            in_asset = trade.base_asset
+            crypto_out_no_fee: RP2Decimal = RP2Decimal(str(transaction[_COST]))
+            conversion_info = f"{trade.quote_info} -> {trade.base_info}"
+        elif transaction[_SIDE] == _SELL:
+            out_asset = trade.base_asset
+            in_asset = trade.quote_asset
+            crypto_out_no_fee = RP2Decimal(str(transaction[_AMOUNT]))
+            conversion_info = f"{trade.base_info} -> {trade.quote_info}"
+        else:
+            raise Exception(f"Internal error: unrecognized transaction side: {transaction[_SIDE]}")
+
+        if transaction[_FEE][_CURRENCY] == out_asset:
+            crypto_fee: RP2Decimal = RP2Decimal(str(transaction[_FEE][_COST]))
+        else:
+            crypto_fee = ZERO
+        crypto_out_with_fee: RP2Decimal = crypto_out_no_fee + crypto_fee
+
+        # Is this a plain buy or a conversion?
+        if trade.quote_asset in self.client.options[_LEGALMONEY]:  # Binance specific param
+            fiat_out_no_fee: RP2Decimal = RP2Decimal(str(transaction[_COST]))
+            fiat_fee: RP2Decimal = RP2Decimal(crypto_fee)
+            spot_price: RP2Decimal = RP2Decimal(str(transaction[_PRICE]))
+
+            out_transaction_list.append(
+                OutTransaction(
+                    plugin=self.__BINANCE_COM,
+                    unique_id=transaction[_ID],
+                    raw_data=json.dumps(transaction),
+                    timestamp=self._rp2timestamp_from_ms_epoch(transaction[_TIMESTAMP]),
+                    asset=out_asset,
+                    exchange=self.__BINANCE_COM,
+                    holder=self.account_holder,
+                    transaction_type=Keyword.SELL.value,
+                    spot_price=str(spot_price),
+                    crypto_out_no_fee=str(crypto_out_no_fee),
+                    crypto_fee=str(crypto_fee),
+                    crypto_out_with_fee=str(crypto_out_with_fee),
+                    fiat_out_no_fee=str(fiat_out_no_fee),
+                    fiat_fee=str(fiat_fee),

```

Comment:
Both crypto fee and fiat fee are assigned in this out-transaction:
* a few lines above (line 1180) `fiat_fee = crypto_fee`: is this the intended behavior (crypto and fiat are the same)?
* the fact that both fiat and crypto fees are assigned is a bit suspicious: typically they are not used together (except in the case of the exchange reporting both and their values don't match). See this FAQ on the topic: https://github.com/eprbell/rp2/blob/main/docs/user_faq.md#how-to-represent-fiat-vs-crypto-transaction-fees.